### PR TITLE
Update bylaws to reflect extended and permanent arch board

### PR DESCRIPTION
**Pull request by gheine**

### CODE REVIEW

File: Structure.md [hunk 1]
```diff
@@ -1,10 +1,9 @@
 General Structure
 -----------------
 
-  - Target 4 members at any given time. Over time may develop as % of engineers or % of teams in Gilt Tech
-  - 24 month term
-  - Members can serve no more than 2 consecutive terms
-  - One member of the board will be the Chair. When the chair rotates off the board, the board will decide its own chair.
+  - The architecture board consists of all principal and distinguished engineers.
+  - One member of the board will be the Chair. The chair is elected on a 12 month term.
+  - Attendance at board meetings is voluntary, however each department should be represented.
```
Comment (chrhicks) [inline Structure.md]: Should we expand upon this? Does it have to be the same person from meeting to meeting?

**Comment by chrhicks** [comment]
:hamburger:

**Comment by mbryzek** [comment]
LGTM

File: Structure.md (hunk 1 above)
Comment (mbryzek) [inline Structure.md]: In earlier discussions, I think we decided the best approach was to just trust the departments to do the best thing. If a dept thinks that means rotating people mtg to mtg, then they should do that. If the dept thinks a consistent attendee is best, do that. If they want to bring all the principals + distinguished to all the meetings, then they do that... e.g. it should just work itself out :)

**Comment by andreyk0** [comment]
:+1:

**Comment by arkitrave** [comment]
Given the new size and structure of the board, and voluntary attendance, do we need to decide on what percentage of members constitutes a quorum for purposes of voting?

**Comment by arkitrave** [comment]
Also, perhaps a clause about the arch board appointing any other members it wants to appoint, since chicks and the discussion about an Ireland member.

**Comment by gheine** [comment]
Not sure I agree on the quorum. By announcing decisions a week in advance should give members enough time to prepare and make sure they are present if they want to have a say. In the case where attendance is continuously low, introducing a quorum will only prevent us from making decisions.

**Comment by gheine** [comment]
Wrt appointment of other members, at least for now (with 12 principal/distinguished engineers) I'd keep the membership at that. Chicks has grandfathered membership and as such the rule doesn't apply to him - besides the fact that imo he should be made principal anyway.

**Comment by gheine** [comment]
How about adding a clause that other members of the tech organization can be invited to board meetings, e.g. because they are expert in a particular area or can provide valuable input?

**Pull request merged**